runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -14665,3 +14665,19 @@ J9::Z::TreeEvaluator::inlineIntegerStringSize(TR::Node* node, TR::CodeGenerator*
 
    return node->setRegister(lengthReg);
    }
+
+TR::Register*
+J9::Z::TreeEvaluator::inlineOnSpinWait(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   // onSpinWait() method calls VM_AtomicSupport::yieldCPU() which is a simple NOP instruction on Z.
+   // Check omr/include_core/AtomicSupport.hpp for the JNI implementation.
+   TR::Instruction* cursor = new (cg->trHeapMemory()) TR::S390NOPInstruction(TR::InstOpCode::NOP, 2, node, cg);
+
+   TR::Compilation *comp = cg->comp();
+   if (comp->getOption(TR_TraceCG))
```

Comment:
Can you move this at the entry of the function - It just seems bit odd reading the code - Cosmetic change.

Reply:
done